### ISSUE TITLE
Fix security vulnerabilities

### ISSUE THREAD

**Issue by pablotolosa-aircury** [elemwave/website#18]
According to the Security tab: https://github.com/elemwave/website/security we have some security issues to deal with

**Comment by pablotolosa-aircury** [comment]
Added the required gems to fix security problems #20

**Comment by cristobaldiaz-aircury** [comment]
There's a solution already to solve the security issues provided by the bot: https://github.com/elemwave/website/pull/8/files.
Please, compare which solution is better and justify why before choosing one.

**Comment by pablotolosa-aircury** [comment]
The commit i pushed resolve the rest of the problems that arent involved with the nokogiri version

**Comment by cristobaldiaz-aircury** [comment]
It would be nice to have the proof that this solves the issues. What command did you run to see if there are no more vulnerabilities?

Moving both back to In Progress

**Comment by pablotolosa-aircury** [comment]
Using the information of this webs as reference:
- https://github.com/envygeeks/jekyll-docker/blob/master/README.md#dependencies
- https://medium.com/@rachelchervin/how-to-audit-your-gems-aca160e91418

Executing the command  "export JEKYLL_VERSION=4.2.0        
docker run --rm \  --volume="$PWD:/srv/jekyll:Z" \  -it jekyll/jekyll:$JEKYLL_VERSION \  bundle update "
when trying to update the gemfile.lock file shows an error related to bundler not being able to find the compatible version for some gems.

![Image](https://github.com/elemwave/website/assets/148042915/098cd2b2-5720-4579-9ffc-25a7e2aa72e0)

On the other hand, we tested  bundle-audit to find the security vulnerabilities. It shows us that the commonmarker version is the problem and we need to update that gem in order to fix the vulnerabilities.

![Image](https://github.com/elemwave/website/assets/148042915/2a224f79-6d56-41ed-8778-d58a24c12faf)

I tried to run a container with a dockerfile and update the gems from there but it was impossible because the bundler wasnt able to find the Gemfile.

We need to find a way to update the Gemfile.lock to fix the vulnerabilities.
